[2017-07-07 15:38:57,815][INFO ][main]
[2017-07-07 15:38:57,828][INFO ][main]Could not initialize class com.tuhanbao.thirdapi.cache.redis.RedisConfig
[2017-07-07 15:38:57,829][WARN ][main]no config file for : ftp
[2017-07-07 15:38:57,842][INFO ][main]
[2017-07-07 15:49:34,464][WARN ][main]no config file for : redis
[2017-07-07 15:49:34,516][WARN ][main]no config file for : ftp
[2017-07-07 15:49:34,529][INFO ][main]
[2017-07-07 15:50:35,285][WARN ][main]no config file for : redis
[2017-07-07 15:50:35,343][WARN ][main]no config file for : ftp
[2017-07-07 15:50:35,357][INFO ][main]
[2017-07-07 15:52:49,800][WARN ][main]no config file for : redis
[2017-07-07 15:52:49,851][WARN ][main]no config file for : ftp
[2017-07-07 15:52:49,863][INFO ][main]
[2017-07-07 16:29:20,034][WARN ][main]no config file for : redis
[2017-07-07 16:29:20,088][WARN ][main]no config file for : ftp
[2017-07-07 16:29:20,101][INFO ][main]
[2017-07-07 16:29:22,406][INFO ][main]java.io.FileInputStream.open(Native Method)
[2017-07-07 16:29:22,406][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:146)
[2017-07-07 16:29:22,406][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:101)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:29:22,406][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:29:22,406][ERROR][main]java.io.FileNotFoundException: E:\git\tuhanbao\demo\config\~$config_debug.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-07-07 16:29:31,284][WARN ][main]no config file for : redis
[2017-07-07 16:29:31,335][WARN ][main]no config file for : ftp
[2017-07-07 16:29:31,348][INFO ][main]
[2017-07-07 16:30:58,945][WARN ][main]no config file for : redis
[2017-07-07 16:30:58,998][WARN ][main]no config file for : ftp
[2017-07-07 16:30:59,011][INFO ][main]
[2017-07-07 16:31:42,198][WARN ][main]no config file for : redis
[2017-07-07 16:31:42,256][WARN ][main]no config file for : ftp
[2017-07-07 16:31:42,269][INFO ][main]
[2017-07-07 16:31:54,813][WARN ][main]no config file for : redis
[2017-07-07 16:31:54,870][WARN ][main]no config file for : ftp
[2017-07-07 16:32:39,816][WARN ][main]no config file for : redis
[2017-07-07 16:32:39,873][WARN ][main]no config file for : ftp
[2017-07-07 16:33:50,720][WARN ][main]no config file for : redis
[2017-07-07 16:33:50,777][WARN ][main]no config file for : ftp
[2017-07-07 16:33:50,790][WARN ][main]no config file for : ds
[2017-07-07 16:34:36,993][WARN ][main]no config file for : redis
[2017-07-07 16:34:37,050][WARN ][main]no config file for : ftp
[2017-07-07 16:34:37,064][WARN ][main]no config file for : ds
[2017-07-07 16:39:04,037][WARN ][main]no config file for : redis
[2017-07-07 16:39:04,090][WARN ][main]no config file for : ftp
[2017-07-07 16:39:04,103][WARN ][main]no config file for : ds
[2017-07-07 16:39:17,446][WARN ][main]no config file for : redis
[2017-07-07 16:39:17,503][WARN ][main]no config file for : ftp
[2017-07-07 16:39:17,516][WARN ][main]no config file for : ds
[2017-07-07 16:40:01,135][WARN ][main]no config file for : redis
[2017-07-07 16:40:01,192][WARN ][main]no config file for : ftp
[2017-07-07 16:40:01,206][WARN ][main]no config file for : ds
[2017-07-07 16:44:04,864][WARN ][main]no config file for : redis
[2017-07-07 16:44:04,921][WARN ][main]no config file for : ftp
[2017-07-07 16:44:04,934][WARN ][main]no config file for : ds
[2017-07-07 16:44:48,094][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,094][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,094][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,094][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,095][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,095][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,095][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,095][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,096][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,096][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,096][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,096][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,112][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,112][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,112][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,112][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,113][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,113][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,113][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,113][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,113][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,113][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,113][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,114][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,114][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,130][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,130][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,130][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,130][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,131][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,131][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,131][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,131][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,131][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,131][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,131][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,131][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,131][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,132][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,132][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,149][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,149][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,149][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,149][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,150][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,150][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,150][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,150][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,151][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,151][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,151][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,151][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,151][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,166][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,166][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,166][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,166][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,167][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,167][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,167][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,167][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,167][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,167][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,167][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,167][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,167][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,167][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,167][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,187][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,187][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,187][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,187][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,188][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,188][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,188][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,188][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,189][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,189][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,189][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,189][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,189][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,189][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,205][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,205][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,205][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,205][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,206][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,206][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,206][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,206][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,206][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,206][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,206][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,206][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,207][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,207][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,222][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,222][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,222][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,222][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,223][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,223][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,223][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,223][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,223][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,223][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,223][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,224][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,224][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,239][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,239][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,239][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,239][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,240][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,240][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,240][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,240][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,241][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,241][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,241][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,241][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,241][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,242][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,257][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,258][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,258][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,258][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,258][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,258][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,258][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,258][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,258][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,258][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,258][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,276][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,276][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,276][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,276][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,277][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,277][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,277][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,277][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,277][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,277][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,278][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,278][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,278][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,278][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,278][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,291][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,291][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,291][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,291][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,292][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,292][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,292][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,292][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,292][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,292][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,293][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,293][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,309][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,309][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,309][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,309][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,310][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,310][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,310][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,310][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,310][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,310][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,310][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,310][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,325][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,325][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,325][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,325][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,326][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,326][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,326][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,326][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,326][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,327][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,327][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,327][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,398][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,398][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,398][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,398][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,398][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,398][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,399][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,399][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,399][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,399][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,399][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,399][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,399][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,399][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,399][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,400][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,400][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,414][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,414][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,414][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,414][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,415][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,415][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,415][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,415][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,415][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,416][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,416][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,432][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,432][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,433][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,433][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,433][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,433][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,433][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,433][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,433][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,446][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,447][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,447][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,447][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,447][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,447][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,448][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,448][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,448][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,448][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,448][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,464][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,465][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,465][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,466][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,466][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,466][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,466][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,466][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,466][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,466][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,483][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,483][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,483][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,484][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,484][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,484][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,484][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,484][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,485][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,485][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,498][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,499][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,499][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,499][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,500][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,500][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,500][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,500][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,500][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,500][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,500][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,500][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,500][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:53,718][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:53,718][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:53,719][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:53,719][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:53,719][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:53,719][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:53,719][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:53,719][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:53,719][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTables(MysqlTableSrcUtil.java:41)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTables(TableSrcUtilFactory.java:44)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getTables(DBUtil.java:152)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:48)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:53,720][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:53,720][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:50)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:53,720][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:53,720][ERROR][main]java.lang.NullPointerException
[2017-07-07 16:46:29,417][WARN ][main]no config file for : redis
[2017-07-07 16:46:29,470][WARN ][main]no config file for : ftp
[2017-07-07 16:46:29,482][WARN ][main]no config file for : ds
[2017-07-07 17:15:56,093][WARN ][main]no config file for : redis
[2017-07-07 17:15:56,149][WARN ][main]no config file for : ftp
[2017-07-07 17:15:56,163][WARN ][main]no config file for : ds
[2017-07-07 17:17:14,809][WARN ][main]no config file for : redis
[2017-07-07 17:17:14,865][WARN ][main]no config file for : ftp
[2017-07-07 17:17:14,879][WARN ][main]no config file for : ds
[2017-07-07 17:18:40,735][WARN ][main]no config file for : redis
[2017-07-07 17:18:40,793][WARN ][main]no config file for : ftp
[2017-07-07 17:18:40,805][WARN ][main]no config file for : ds
[2017-07-07 17:19:51,941][WARN ][main]no config file for : redis
[2017-07-07 17:19:51,996][WARN ][main]no config file for : ftp
[2017-07-07 17:19:52,009][WARN ][main]no config file for : ds
[2017-08-29 23:31:10,376][WARN ][main]no config file for : redis
[2017-08-29 23:31:10,861][WARN ][main]no config file for : ftp
[2017-08-29 23:31:11,077][WARN ][main]no config file for : ds
[2017-08-29 23:33:05,196][WARN ][main]no config file for : redis
[2017-08-29 23:33:05,258][WARN ][main]no config file for : ftp
[2017-08-29 23:33:05,273][WARN ][main]no config file for : ds
[2017-08-29 23:34:48,173][WARN ][main]no config file for : redis
[2017-08-29 23:34:48,226][WARN ][main]no config file for : ftp
[2017-08-29 23:34:48,240][WARN ][main]no config file for : ds
[2017-08-29 23:34:53,153][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:53,154][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:53,154][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:53,154][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:53,154][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:53,154][INFO ][main]com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1074)
[2017-08-29 23:34:53,154][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2103)
[2017-08-29 23:34:53,155][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-29 23:34:53,155][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-29 23:34:53,155][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:53,155][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:53,155][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:53,156][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:53,156][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:53,156][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-29 23:34:53,156][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-29 23:34:53,156][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-29 23:34:53,156][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-29 23:34:53,157][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-29 23:34:53,157][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-29 23:34:53,157][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-29 23:34:53,157][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-29 23:34:53,157][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-29 23:34:53,157][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-29 23:34:53,158][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-29 23:34:53,158][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-29 23:34:53,158][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-29 23:34:53,158][INFO ][main]Test.main(Test.java:18)
[2017-08-29 23:34:53,158][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

Last packet sent to the server was 0 ms ago.
[2017-08-29 23:34:55,217][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:55,217][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:55,218][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:55,218][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:55,218][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:55,218][INFO ][main]com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1074)
[2017-08-29 23:34:55,219][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2103)
[2017-08-29 23:34:55,219][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-29 23:34:55,219][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-29 23:34:55,220][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:55,220][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:55,220][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:55,220][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:55,221][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:55,221][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-29 23:34:55,221][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-29 23:34:55,221][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-29 23:34:55,222][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-29 23:34:55,222][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-29 23:34:55,222][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-29 23:34:55,222][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-29 23:34:55,223][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-29 23:34:55,223][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-29 23:34:55,223][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-29 23:34:55,223][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-29 23:34:55,224][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-29 23:34:55,224][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-29 23:34:55,224][INFO ][main]Test.main(Test.java:18)
[2017-08-29 23:34:55,225][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

Last packet sent to the server was 0 ms ago.
[2017-08-29 23:34:57,232][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:57,232][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:57,233][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:57,233][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:57,233][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:57,233][INFO ][main]com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1074)
[2017-08-29 23:34:57,233][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2103)
[2017-08-29 23:34:57,233][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-29 23:34:57,233][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-29 23:34:57,233][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:57,234][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:57,234][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:57,234][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:57,234][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:57,234][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-29 23:34:57,235][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-29 23:34:57,235][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-29 23:34:57,235][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-29 23:34:57,235][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-29 23:34:57,235][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-29 23:34:57,236][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-29 23:34:57,236][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-29 23:34:57,236][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-29 23:34:57,236][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-29 23:34:57,236][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-29 23:34:57,237][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-29 23:34:57,237][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-29 23:34:57,237][INFO ][main]Test.main(Test.java:18)
[2017-08-29 23:34:57,237][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

Last packet sent to the server was 0 ms ago.
[2017-08-29 23:34:59,245][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:59,245][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:59,245][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:59,246][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:59,246][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:59,246][INFO ][main]com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1074)
[2017-08-29 23:34:59,246][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2103)
[2017-08-29 23:34:59,246][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-29 23:34:59,247][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-29 23:34:59,247][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:34:59,247][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:34:59,247][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:34:59,247][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:34:59,248][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:34:59,248][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-29 23:34:59,248][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-29 23:34:59,248][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-29 23:34:59,248][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-29 23:34:59,249][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-29 23:34:59,249][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-29 23:34:59,249][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-29 23:34:59,249][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-29 23:34:59,249][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-29 23:34:59,250][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-29 23:34:59,250][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-29 23:34:59,250][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-29 23:34:59,250][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-29 23:34:59,250][INFO ][main]Test.main(Test.java:18)
[2017-08-29 23:34:59,251][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

Last packet sent to the server was 0 ms ago.
[2017-08-29 23:35:01,256][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:35:01,256][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:35:01,257][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:35:01,257][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:35:01,257][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:35:01,257][INFO ][main]com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:1074)
[2017-08-29 23:35:01,257][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2103)
[2017-08-29 23:35:01,258][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-29 23:35:01,258][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-29 23:35:01,258][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-29 23:35:01,258][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-29 23:35:01,259][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-29 23:35:01,259][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-29 23:35:01,259][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-29 23:35:01,259][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-29 23:35:01,260][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-29 23:35:01,260][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-29 23:35:01,260][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-29 23:35:01,260][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-29 23:35:01,260][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-29 23:35:01,260][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-29 23:35:01,261][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-29 23:35:01,261][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-29 23:35:01,261][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-29 23:35:01,261][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-29 23:35:01,261][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-29 23:35:01,261][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-29 23:35:01,262][INFO ][main]Test.main(Test.java:18)
[2017-08-29 23:35:01,262][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

Last packet sent to the server was 0 ms ago.
[2017-08-31 23:04:58,424][WARN ][main]no config file for : redis
[2017-08-31 23:04:58,909][WARN ][main]no config file for : ftp
[2017-08-31 23:04:59,180][WARN ][main]no config file for : ds
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,450][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,451][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,451][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,451][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,451][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,451][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,451][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,451][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,451][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,451][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,451][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,451][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,451][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,451][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,451][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,451][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,452][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,452][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,452][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,452][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,452][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,452][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,468][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,468][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,468][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,468][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,469][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,469][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,469][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,469][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,469][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,469][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,470][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,470][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,470][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,470][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,473][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,473][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,474][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,474][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,474][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,474][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,474][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,474][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,474][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,474][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,474][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,474][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,474][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,475][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,475][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,475][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,475][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,475][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,475][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,475][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,475][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,477][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,478][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,478][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,478][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,478][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,478][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,478][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,479][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,479][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,479][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,479][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,482][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,482][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,482][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,482][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,483][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,483][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,483][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,483][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,483][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,483][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,483][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,483][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,483][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,486][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,486][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,486][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,487][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,487][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,487][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,487][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,487][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,487][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,487][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,487][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,487][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,487][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,490][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,490][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,490][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,490][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,491][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,491][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,491][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,491][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,491][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,491][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,491][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,491][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,491][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,491][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,492][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,492][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,492][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,492][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,492][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,492][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,492][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,492][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,494][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,495][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,495][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,495][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,495][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,495][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,495][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,495][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,496][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,496][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,496][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,498][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,498][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,498][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,498][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,499][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,499][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,499][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,499][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,499][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,499][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,499][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,499][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,499][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,499][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,502][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,502][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,503][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,503][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,503][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,503][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,503][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,503][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,503][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,503][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,503][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,503][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,503][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,507][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,507][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,508][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,508][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,508][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,508][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,508][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,508][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,508][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,508][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,508][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,509][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,509][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,509][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,512][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,512][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,512][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,512][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,512][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,512][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,512][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,512][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,512][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,512][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,512][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-08-31 23:05:03,512][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-08-31 23:05:03,513][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-08-31 23:05:03,513][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-08-31 23:05:03,513][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-08-31 23:05:03,513][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,513][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,513][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-08-31 23:05:03,516][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-08-31 23:05:03,516][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2017-08-31 23:05:03,516][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-08-31 23:05:03,516][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:423)
[2017-08-31 23:05:03,516][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-08-31 23:05:03,517][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-08-31 23:05:03,517][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-08-31 23:05:03,517][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:664)
[2017-08-31 23:05:03,517][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:208)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTables(MysqlTableSrcUtil.java:41)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTables(TableSrcUtilFactory.java:44)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getTables(DBUtil.java:152)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:48)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,517][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,517][ERROR][main]java.sql.SQLException: Access denied for user 'cfcrm_user'@'localhost' (using password: YES)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:50)
[2017-08-31 23:05:03,517][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-08-31 23:05:03,517][INFO ][main]Test.main(Test.java:18)
[2017-08-31 23:05:03,517][ERROR][main]java.lang.NullPointerException
[2017-08-31 23:05:25,217][WARN ][main]no config file for : redis
[2017-08-31 23:05:25,281][WARN ][main]no config file for : ftp
[2017-08-31 23:05:25,297][WARN ][main]no config file for : ds
[2017-08-31 23:08:32,405][WARN ][main]no config file for : redis
[2017-08-31 23:08:32,459][WARN ][main]no config file for : ftp
[2017-08-31 23:08:32,472][WARN ][main]no config file for : ds
[2017-08-31 23:19:32,205][WARN ][main]no config file for : redis
[2017-08-31 23:19:32,276][WARN ][main]no config file for : ftp
[2017-08-31 23:19:32,290][WARN ][main]no config file for : ds
[2017-08-31 23:22:18,465][WARN ][main]no config file for : redis
[2017-08-31 23:22:18,522][WARN ][main]no config file for : ftp
[2017-08-31 23:22:18,534][WARN ][main]no config file for : ds
[2017-09-01 21:10:55,924][WARN ][main]no config file for : redis
[2017-09-01 21:10:55,977][WARN ][main]no config file for : ftp
[2017-09-01 21:10:55,990][WARN ][main]no config file for : ds
[2017-09-01 21:10:57,243][INFO ][main]java.io.FileInputStream.open0(Native Method)
[2017-09-01 21:10:57,243][INFO ][main]java.io.FileInputStream.open(FileInputStream.java:195)
[2017-09-01 21:10:57,243][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:138)
[2017-09-01 21:10:57,243][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:93)
[2017-09-01 21:10:57,243][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-09-01 21:10:57,243][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-09-01 21:10:57,244][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-09-01 21:10:57,244][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-09-01 21:10:57,244][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-09-01 21:10:57,244][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-09-01 21:10:57,244][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-09-01 21:10:57,244][INFO ][main]Test.main(Test.java:18)
[2017-09-01 21:10:57,244][ERROR][main]java.io.FileNotFoundException: D:\workspace\git\tuhanbao\crm\config\~$crm.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-09-01 21:11:09,147][WARN ][main]no config file for : redis
[2017-09-01 21:11:09,199][WARN ][main]no config file for : ftp
[2017-09-01 21:11:09,212][WARN ][main]no config file for : ds
[2017-09-01 22:50:27,189][WARN ][main]no config file for : redis
[2017-09-01 22:50:27,243][WARN ][main]no config file for : ftp
[2017-09-01 22:50:27,258][WARN ][main]no config file for : ds
[2017-09-01 23:44:12,790][WARN ][main]no config file for : redis
[2017-09-01 23:44:12,843][WARN ][main]no config file for : ftp
[2017-09-01 23:44:12,858][WARN ][main]no config file for : ds
[2017-09-01 23:44:14,745][INFO ][main]java.io.FileInputStream.open(Native Method)
[2017-09-01 23:44:14,745][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:146)
[2017-09-01 23:44:14,746][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:101)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-09-01 23:44:14,746][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-09-01 23:44:14,746][INFO ][main]Test.main(Test.java:18)
[2017-09-01 23:44:14,746][ERROR][main]java.io.FileNotFoundException: D:\workspace\git\tuhanbao\crm\config\~$config.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-09-02 00:44:51,482][WARN ][main]no config file for : redis
[2017-09-02 00:44:51,536][WARN ][main]no config file for : ftp
[2017-09-02 00:44:51,551][WARN ][main]no config file for : ds
[2017-09-02 00:44:51,553][INFO ][main]
[2017-09-02 00:45:48,270][WARN ][main]no config file for : redis
[2017-09-02 00:45:48,331][WARN ][main]no config file for : ftp
[2017-09-02 00:45:48,347][WARN ][main]no config file for : ds
[2017-09-02 00:45:53,105][INFO ][main]
[2017-09-02 00:46:46,315][WARN ][main]no config file for : redis
[2017-09-02 00:46:46,374][WARN ][main]no config file for : ftp
[2017-09-02 00:46:46,390][WARN ][main]no config file for : ds
[2017-09-02 00:46:55,423][INFO ][main]
[2017-09-02 00:48:15,693][WARN ][main]no config file for : redis
[2017-09-02 00:48:15,751][WARN ][main]no config file for : ftp
[2017-09-02 00:48:15,767][WARN ][main]no config file for : ds
[2017-09-02 00:48:30,296][INFO ][main]
[2017-09-02 00:49:26,116][WARN ][main]no config file for : redis
[2017-09-02 00:49:26,181][WARN ][main]no config file for : ftp
[2017-09-02 00:49:26,197][WARN ][main]no config file for : ds
[2017-09-02 00:49:47,690][INFO ][main]
[2017-09-02 00:49:56,153][WARN ][main]no config file for : redis
[2017-09-02 00:49:56,214][WARN ][main]no config file for : ftp
[2017-09-02 00:49:56,230][WARN ][main]no config file for : ds
[2017-09-02 00:49:58,964][INFO ][main]
[2017-09-02 00:50:04,115][WARN ][main]no config file for : redis
[2017-09-02 00:50:04,174][WARN ][main]no config file for : ftp
[2017-09-02 00:50:04,189][WARN ][main]no config file for : ds
[2017-09-02 00:50:38,170][WARN ][main]no config file for : redis
[2017-09-02 00:50:38,229][WARN ][main]no config file for : ftp
[2017-09-02 00:50:38,244][WARN ][main]no config file for : ds
[2017-09-02 00:51:48,972][WARN ][main]no config file for : redis
[2017-09-02 00:51:49,030][WARN ][main]no config file for : ftp
[2017-09-02 00:51:49,046][WARN ][main]no config file for : ds
[2017-09-02 00:52:01,001][ERROR][main]
[2017-09-02 00:52:03,286][WARN ][main]no config file for : redis
[2017-09-02 00:52:03,372][WARN ][main]no config file for : ftp
[2017-09-02 00:52:03,392][WARN ][main]no config file for : ds
[2017-09-02 00:52:46,388][WARN ][main]no config file for : redis
[2017-09-02 00:52:46,449][WARN ][main]no config file for : ftp
[2017-09-02 00:52:46,465][WARN ][main]no config file for : ds
[2017-09-04 22:53:23,218][WARN ][main]no config file for : redis
[2017-09-04 22:53:23,331][WARN ][main]no config file for : ftp
[2017-09-04 22:53:23,468][WARN ][main]no config file for : ds
[2017-09-04 22:57:28,533][WARN ][main]no config file for : redis
[2017-09-04 22:57:28,602][WARN ][main]no config file for : ftp
[2017-09-04 22:57:28,620][WARN ][main]no config file for : ds
[2017-09-06 23:37:01,006][WARN ][main]no config file for : redis
[2017-09-06 23:37:01,210][WARN ][main]no config file for : ftp
[2017-09-06 23:37:01,333][WARN ][main]no config file for : ds
[2017-09-06 23:37:03,537][INFO ][main]java.io.FileInputStream.open(Native Method)
[2017-09-06 23:37:03,537][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:146)
[2017-09-06 23:37:03,537][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:101)
[2017-09-06 23:37:03,537][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-09-06 23:37:03,537][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-09-06 23:37:03,537][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-09-06 23:37:03,537][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-09-06 23:37:03,538][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-09-06 23:37:03,538][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-09-06 23:37:03,538][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-09-06 23:37:03,538][INFO ][main]Test.main(Test.java:18)
[2017-09-06 23:37:03,538][ERROR][main]java.io.FileNotFoundException: D:\workspace\git\tuhanbao\crm\config\~$crm.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-09-06 23:37:17,346][WARN ][main]no config file for : redis
[2017-09-06 23:37:17,405][WARN ][main]no config file for : ftp
[2017-09-06 23:37:17,420][WARN ][main]no config file for : ds
[2017-09-06 23:40:56,325][WARN ][main]no config file for : redis
[2017-09-06 23:40:56,381][WARN ][main]no config file for : ftp
[2017-09-06 23:40:56,400][WARN ][main]no config file for : ds
[2017-09-06 23:43:15,115][WARN ][main]no config file for : redis
[2017-09-06 23:43:15,171][WARN ][main]no config file for : ftp
[2017-09-06 23:43:15,188][WARN ][main]no config file for : ds
[2017-09-06 23:45:00,274][WARN ][main]no config file for : redis
[2017-09-06 23:45:00,356][WARN ][main]no config file for : ftp
[2017-09-06 23:45:00,379][WARN ][main]no config file for : ds
[2017-09-06 23:46:15,652][WARN ][main]no config file for : redis
[2017-09-06 23:46:15,708][WARN ][main]no config file for : ftp
[2017-09-06 23:46:15,724][WARN ][main]no config file for : ds
[2017-09-06 23:46:28,144][WARN ][main]no config file for : redis
[2017-09-06 23:46:28,200][WARN ][main]no config file for : ftp
[2017-09-06 23:46:28,216][WARN ][main]no config file for : ds
[2017-09-06 23:47:47,509][WARN ][main]no config file for : redis
[2017-09-06 23:47:47,563][WARN ][main]no config file for : ftp
[2017-09-06 23:47:47,579][WARN ][main]no config file for : ds
[2017-09-06 23:50:37,097][WARN ][main]no config file for : redis
[2017-09-06 23:50:37,155][WARN ][main]no config file for : ftp
[2017-09-06 23:50:37,173][WARN ][main]no config file for : ds
[2017-09-13 21:00:17,038][WARN ][main]no config file for : redis
[2017-09-13 21:00:17,105][WARN ][main]no config file for : ftp
[2017-09-13 21:00:17,122][WARN ][main]no config file for : ds
[2017-09-13 21:00:19,231][INFO ][main]java.io.FileInputStream.open(Native Method)
[2017-09-13 21:00:19,231][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:146)
[2017-09-13 21:00:19,231][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:101)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-09-13 21:00:19,231][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-09-13 21:00:19,231][INFO ][main]Test.main(Test.java:18)
[2017-09-13 21:00:19,231][ERROR][main]java.io.FileNotFoundException: D:\workspace\git\tuhanbao\crm\config\~$crm.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-09-13 21:02:09,616][WARN ][main]no config file for : redis
[2017-09-13 21:02:09,679][WARN ][main]no config file for : ftp
[2017-09-13 21:02:09,699][WARN ][main]no config file for : ds
[2017-09-14 00:05:14,258][WARN ][main]no config file for : redis
[2017-09-14 00:05:14,319][WARN ][main]no config file for : ftp
[2017-09-14 00:05:14,336][WARN ][main]no config file for : ds
[2017-09-23 16:36:47,992][WARN ][main]no config file for : redis
[2017-09-23 16:36:48,065][WARN ][main]no config file for : ftp
[2017-09-23 16:36:48,081][WARN ][main]no config file for : ds
[2017-09-23 16:36:50,000][INFO ][main]java.io.FileInputStream.open(Native Method)
[2017-09-23 16:36:50,000][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:146)
[2017-09-23 16:36:50,000][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:101)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-09-23 16:36:50,000][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-09-23 16:36:50,000][INFO ][main]Test.main(Test.java:18)
[2017-09-23 16:36:50,000][ERROR][main]java.io.FileNotFoundException: D:\workspace\git\tuhanbao\crm\config\~$config.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-09-23 16:37:21,442][WARN ][main]no config file for : redis
[2017-09-23 16:37:21,497][WARN ][main]no config file for : ftp
[2017-09-23 16:37:21,512][WARN ][main]no config file for : ds
